statsd-exporter/version.txt

```diff
@@ -1 +1 @@
-0.23.1
+0.23.2
```

Comment:
We're rebuilding 0.23.1 https://github.com/astronomer/ap-vendor/blob/74cfcf4a1e3dbb989245f936dbf7808e62645c0b/statsd-exporter/Dockerfile#L17 so this should be:

```suggestion
0.23.1-1
```